src/use_case/vote_out/VoteOutInteractor.java

```diff
@@ -1,4 +1,57 @@
 package use_case.vote_out;
 
+import entity.Game;
+import entity.PromptGenerator;
+import use_case.ChatAPIAccessInterface;
+
 public class VoteOutInteractor implements VoteOutInputBoundary {
+    final VoteOutDataAccessInterface gameDataAccessObject;
+    final ChatAPIAccessInterface storyDataAccessObject;
+    final VoteOutOutputBoundary userPresenter;
+
+    public VoteOutInteractor(VoteOutDataAccessInterface gameDataAccessObject, ChatAPIAccessInterface storyDataAccessObject, VoteOutOutputBoundary userPresenter) {
```

Comment:
One thing I was thinking was that for the constructor, instead of keeping a gameDataAccessObject and storyDataAccessObject, we can just have a PromptGenerator attribute and a Game attribute by constructing the two in the VoteOutInteractor method. 

Let me know what you think, I'll probably do that for my use case because I feel like it will make the voteOutPlayer more concise. It's more of a nitpick more than anything.

Reply:
we can't do that because you need to save changes to the game to the data access object and not the entity that we create, and also you also need both the prompt generator and the story data access object to create the story (look through the code in the voteOutPlayer method)

Reply:
Basically to clarify, if you just change the game entity you're not actually going to end up changing anything in the csv file so you need to keep the data access object as an attribute. I made a save(game) method that will update the csv file. As for the storydataccessobject yeah you just need it see line 44.

Reply:
Okay yeah makes sense, I realized that when writing the KillVillager use case. For the game and promptGenerator, I still pulled them out as attributes instead of having them in the killVillager method, let me know if you want to do that as well for the VotePlayer class because I think our code should look similar for ease of understanding when the TA reads our classes.

Reply:
Ah okay that makes sense I’ll do it tomorrow